src/messages.rs

```diff
@@ -250,15 +306,24 @@ impl SignedMessage {
         &self.content
     }
 
-    /// The `PublicId` associated with the signed message
-    pub fn public_id(&self) -> &PublicId {
-        &self.public_id
-    }
-
     /// The priority Crust should send this message with.
     pub fn priority(&self) -> u8 {
         self.content.priority()
     }
+
+    /// Returns whether there are enough valid signatures from the sender.
+    pub fn is_fully_signed(&self) -> bool {
+        if self.content.src.is_client() {
```

Comment:
Why do we have this separate check (with a more stringent condition requiring `len() == 1`) for clients?